docs/customization/custom-scripts.md

```diff
@@ -304,6 +304,7 @@ A particular object within NetBox. Each ObjectVar must specify a particular mode
 
 * `model` - The model class
 * `query_params` - A dictionary of query parameters to use when retrieving available options (optional)
+* `context` - A custom dictionary mapping template context variables to fields, used when rendering `<option>` elements within the dropdown menu (optional)
```

Comment:
Should we document here the current choices?:
Value
Label
Disabled indicator
Description
Depth in hierarchy (for indentation)
Parent object

Reply:
Yeah, I was just pondering _where_ we should document these. We'll undoubtedly want to retain a reference for DynamicModelChoiceField more generally in addition to ObjectVar.